cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/OutputsTransformer.java

```diff
@@ -30,6 +31,12 @@
  */
 public class OutputsTransformer extends AbstractOutputsTransformer implements Transformer<List<Object>, List<Output>> {
 
+    private static final List<String> ACTIVITY_OUTPUTS_KNOWN_KEYS =
+            Arrays.asList(SENSITIVE_KEY, VALUE_KEY, SEQ_OUTPUT_ROBOT_KEY);
+
+    protected List<String> getKnownKeys() {
```

Comment:
Why do we need this? Can't the SEQ_OUTPUT_ROBOT_KEY be added to the abstract transformer?

Reply:
Because we don't want to allow this property in the "publish" section of a step. This transformer handles the flow/activity outputs.